### PR TITLE
Adds engineering + science crates

### PR DESCRIPTION
**Pull request by nachomeep**
Also enables them for engineering/robotics equipment. 
image

Ported from https://github.com/d3athrow/vgstation13

### CODE REVIEW

**Comment by nachomeep** [comment]
Note that even hydroponics has their own crate style already.

**Comment by Anewbe** [comment]
The issue isn't so much with the crates themselves, as much as it is with the sprites in the image. The engineering ones especially are a bit of an eyesore. I think this'd be much better if the departmental crates just had a colored line, like the mentioned hydroponics crate, because they don't really feel like they fit with the rest of the crate sprites.

**Comment by nachomeep** [comment]
Fair enough. What's the opinion on the scicrates, then? I actually really like those ones.

**Comment by Anewbe** [comment]
Those seem fine, to me.

**Comment by uraniummeltdown** [comment]
the engineering ones are actually from tgstation

**Comment by nachomeep** [comment]
So they are. Thank you. I'll be replacing them anyway so it doesn't matter, though.

**Comment by nachomeep** [comment]
New engicrates: image

File: code/datums/supplypacks/atmospherics.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	name = "Inflatable barriers"
 	contains = list(/obj/item/weapon/storage/briefcase/inflatable = 3)
 	cost = 20
-	containertype = /obj/structure/closet/crate
+	containertype = /obj/structure/closet/crate/secure/engineering
```
Comment (Anewbe) [inline code/datums/supplypacks/atmospherics.dm]: Not sure this warrants a secure crate.

Comment (nachomeep) [inline code/datums/supplypacks/atmospherics.dm]: I don't actually remember why I made it secure, you're right.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -283,7 +283,7 @@
 /obj/structure/closet/crate/rcd
 	name = "\improper RCD crate"
 	desc = "A crate with rapid construction device."
-	icon_state = "crate"
+	icon_state = "engi_crate"
 	icon_opened = "crateopen"
 	icon_closed = "crate"
```
Comment (Anewbe) [inline code/game/objects/structures/crates_lockers/crates.dm]: You sure these other two lines don't need to be updated too?

Comment (Anewbe) [inline code/game/objects/structures/crates_lockers/crates.dm]: Think these other two lines might need to be tweaked a bit.

Comment (nachomeep) [inline code/game/objects/structures/crates_lockers/crates.dm]: ...oh, right.

**Pull request merged**